modules/crypt.cpp

```diff
@@ -200,6 +332,41 @@ class CCryptMod : public CModule {
     }
 
     EModRet OnPrivNotice(CNick& Nick, CString& sMessage) override {
+        CString sCommand = sMessage.Token(0);
+        CString sOtherPubKey = sMessage.Token(1);
+
+        if ((sCommand.Equals("DH1080_INIT") || sCommand.Equals("DH1080_INIT_CBC")) && !sOtherPubKey.empty()) {
```

Comment:
Are you sure that both the normal INIT and the INIT_CBC should be handled the same? Whoever added (or removed) a CBC suffix must have had a reason for it (CBC: cypher block chaining, a not-really-secure but way too common scheme)